### PR TITLE
Add preview

### PR DESCRIPTION
**Pull request by vladimirze**
Adds preview

### CODE REVIEW

**Comment by vladimirze** [comment]
I think this will also should allow sharing image thru whatsapp even though the canvas is too big to fit the screen.

File: src/index.js [hunk 1]
```diff
@@ -18,30 +18,25 @@ function initHealthStatement(form, signaturePad) {
...
-    Storage.save({
-      ...formValues,
-      signature: signaturePad.toData(),
-    });
-  } else {
-    Storage.clear();
-  }
-
   const statement = new HealthStatement(formValues);
   statement.addSignature(signature, date);
```
Comment (shevron) [inline src/index.js:22]: You moved setting the date into the constructor, so I think it is not really needed here anymore

File: src/index.js [hunk 2]
```diff
@@ -55,6 +50,20 @@ document.addEventListener('DOMContentLoaded', () => {
...
+    } else {
+      Storage.clear();
+    }
+    downloadHealthStatement(statement);
   });
+
+  const preview = () => createHealthStatement(form, sigField, signature).preview(previewContainer);
+  preview();
+  form.addEventListener('focusout', preview);
+  sigField.addEventListener('mouseout', preview);
```
Comment (shevron) [inline src/index.js:68]: I think there is a special `signature_pad` event that we can tie into here

**Comment by vladimirze** [comment]
I added minimalistic overlay 🙂

**Pull request merged**